src/components/sidebar/helpers/add-nav-item-meta-data.ts

```diff
@@ -54,7 +54,7 @@ export const addNavItemMetaData = (
 			}
 
 			// Found an `EnrichedLinkNavItem` object
-			if (item.hasOwnProperty('path')) {
+			if (item.hasOwnProperty('fullPath')) {
```

Comment:
We compare on the `fullPath` here, so switched to detect `fullPath` rather than `path` (noticed this while stubbing this work in).